apps/controllerx/devices/ikea.py

```diff
@@ -47,12 +49,16 @@ def get_deconz_actions_mapping(self):
     def get_zha_actions_mapping(self):
         return {
             "toggle": Light.TOGGLE,
+            "move_to_level_with_on_off_254_0": Light.ON_FULL_BRIGHTNESS,
+###         "press_2_0_0": SET A DEFULT COLOR TEMP?
```

Comment:
You said the following:

> A long press generates:
toggle right away, followed by release then move_to_level_with_on_off_254_0 then press_2_0_0 all in quick succession after the center button is held for 3 seconds.

We have the same problem that I described before, the "toggle" action fires first, so it is not useful to use the hold center button.

```suggestion
```